core/src/ldml/C7043_ldml.md

```diff
@@ -439,10 +406,10 @@ For each key:
 |--------------|-----------|---------------------------------------------|
 |       0      | extend    | 0: `to` is a char, 1: `to` is a string      |
 |       1      | gap       | 1 if the key is a gap                       |
-|       2      | transform | 1 if the key is transform=no                |
 
 - `to`: If `extend` is 0, `to` is a UTF-32LE codepoint. If `extend` is 1, `to`
```

Comment:
I think cleanup of `to` to `output` etc in KMX+ to match the XML names should happen sooner rather than later, next PR?

Reply:
OK next PR